### PR TITLE
Change default debug Kconfig for bootloader

### PR DESCRIPTION
**Pull request by sigvartmh**
Since the RTT logging is unstable when swapping to a new image UART should be the default
debug configuration for debug in the bootloader for now.

Signed-off-by: Sigvart Hovland <sigvart.hovland@nordicsemi.no>

### CODE REVIEW

**Comment by lemrey** [comment]
Looks good to me, but I would reword the commit message to specify that you are changing the debug output port, rather than the "debug Kconfig".

**Comment by sigvartmh** [comment]
UI fault didn't mean to close it. Good idea rephrased the commit message.

**Comment by lemrey** [comment]
@sigvartmh Have you set your repository to public?

**Comment by sigvartmh** [comment]
Haven't I'll see if I can fix it if not I'll just push the branch to a public and create a new PR
